isis/tests/TestUtilities.cpp

```diff
@@ -250,4 +250,33 @@ namespace Isis {
         ::testing::Field(&csm::EcefCoord::z, ::testing::DoubleNear(expected.z, 0.0001))
     );
   }
+
+  // Writes binary kernels to the data area. Unsure of the best way to handle
+  // clean up. Didn't want to dive into the rabbit hole of C++ alternatives
+  // to python yeild statements
+  QString generateBinaryKernels(QVector<QString> kernelList) {
+    QString binaryKernelList("(");
+    for (QString kernel : kernelList) {
+      FileName file(kernel);
+      QString kernelExtension = file.extension();
+      QString pathToBinaryKernel = file.path() + "/" + file.baseName() + "." + kernelExtension.replace('x', 'b');
+      FileName binaryFile(pathToBinaryKernel);
+
+      if (kernelExtension.contains("x") && !binaryFile.fileExists()) {
+        QString path = file.expanded();
+        QString command = "tobin " + path;
```

Comment:
Where are the executables kept? I recall these being pretty standalone and shipped separate from the c lib?

Reply:
They are distributed with the c library. There's a handful of special case ones like ckslicer that are not, but tobin is.